### DISCUSSION TITLE
Pint 0.22rc1 is out

### DISCUSSION THREAD

**Discussion by hgrecco** [hgrecco/pint#1780] [Announcements]
This is the first release candidate for Pint 0.22. This version focus on improving typing support. Notice that this version requires  Python 3.9+ and NumPy 1.21+ as per [NEP 29](https://numpy.org/neps/nep-0029-deprecation_policy.html).

If there are no show-stoppers, Pint 0.22 will be released on May 22nd, 2023.

Maintainers of downstream projects, please take a look at this [dashboard](https://github.com/hgrecco/pint/blob/master/downstream_status.md)

**Comment by dopplershift** [comment]
Currently the changes in #1660 have broken all of MetPy's xarray + pint testing. Is there any way we get can a solution for that? That's keeping us from running >= 0.21.

**Comment by hgrecco** [comment]
@dopplershift We will not release Pint 0.22 until major downstream projects that reported problems are running. (BTW, what do you think of this: https://github.com/hgrecco/pint/blob/master/downstream_status.md)

I would also suggest that we release 0.21.1 fixing this. @jules-ch What is your take on this?

**Reply by dopplershift** [reply]
I like it. I just opened #1782.

**Comment by hgrecco** [comment]
Just released 0.21.1rc1 and 0.22rc2 Let me know how are things going.

**Reply by dopplershift** [reply]
MetPy CI is now passing when running against Pint master branch.

**Comment by hgrecco** [comment]
0.21.1 and 0.22 is out.
